admin_manual/installation/example_centos.rst

```diff
@@ -21,8 +21,26 @@ Apache
 ::
 
     dnf install -y httpd
+    
+Create a virtualhost in ``/etc/httpd/conf.d/nextcloud.conf`` and add the following content to it:
+
+::
+
+  <VirtualHost *:80>
+    DocumentRoot /var/www/html/nextcloud/
+    ServerName  your.server.com
+
+    <Directory /var/www/html/nextcloud/>
+      Require all granted
+      AllowOverride All
+      Options FollowSymLinks MultiViews
+
+      <IfModule mod_dav.c>
+        Dav off
+      </IfModule>
 
-See :ref:`apache_configuration_label` for details.
```

Comment:
I would keep this to provide further documentation as well.

Reply:
good point